### PR TITLE
fix #8: Init -> create files in summary.md

### LINKED ISSUE #8: Init -> create files in summary.md
When you use `mdbook init` and `SUMMARY.md` already exists:

Parse `SUMMARY.md` and create the files.

For example, `SUMMARY.md` contains:

```
# Summary

- [Chapter 1](chapter_1.md)
- [Chapter 2](directory/chapter_2.md)
```

Create
```
src/
├── chapter_1.md
├── directory
│   └── chapter_2.md
└── SUMMARY.md
```

### PR DESCRIPTION
**Pull request by FuGangqiang**

### CODE REVIEW

File: src/book/mdbook.rs [hunk 1]
```diff
@@ -92,40 +92,53 @@ impl MDBook {
             output!("{:?} created", self.config.get_root());
         }
 
-        let dest = self.config.get_dest();
-        let src = self.config.get_src();
+        {
```
Comment (azerupi) [inline src/book/mdbook.rs]: What is the reason behind wrapping this part in a block? :)

**Comment by azerupi** [comment]
Nice ! Thanks :)

File: src/book/mdbook.rs (hunk 1 above)
Comment (FuGangqiang) [inline src/book/mdbook.rs]: `self.parse_summary()` mut borrow `self`,  `self.config` immut borrow `self`

Comment (azerupi) [inline src/book/mdbook.rs]: Very well, thanks for this PR !

**Pull request merged**